velox/common/file/FileSystems.cpp

```diff
@@ -100,6 +101,23 @@ class LocalFileSystem : public FileSystem {
     }
   }
 
+  bool exist(std::string_view path) override {
+    auto file =
```

Comment:
The logic for removing kFileScheme repeats in multiple places. Consider extracting a helper method to avoid copy-paste.

```
std::string_view stripFileScheme(std::string_view path);
```